docs/Navigation.md

```diff
@@ -134,4 +134,4 @@ class MyScene extends React.Component {
 }
 ```
 
-Check out the [`NavigatorIOS` reference docs](docs/navigatorios.html) to learn more about this component.
+Check out the [`NavigatorIOS` reference docs](docs/navigation.html) to learn more about this component.
```

Comment:
Navigation iOS page no longer exists. I updated the docs to use the correct page. Not sure what you are asking.